ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -328,6 +328,7 @@ keys:
         name:
           type: str
           description: Access-list Name
+          convert_types: [ int ]
```

Comment:
You should not change this file manually. You need to change the fragments which will then be joined (by pre-commit) to this file.

Reply:
Ack - moved the changes to the fragments